src/tool/subcommands/api_cmd/api_compare_tests.rs

```diff
@@ -443,7 +448,57 @@ fn chain_tests_with_tipset<DB: Blockstore>(
             tipset.epoch(),
             Default::default(),
         ))?),
-        RpcTest::identity(ChainGetTipSet::request((tipset.key().clone().into(),))?),
+        RpcTest::identity(ChainGetTipSet::request((tipset.key().into(),))?),
+        RpcTest::identity(ChainGetTipSet::request((None.into(),))?)
+            .policy_on_rejected(PolicyOnRejected::PassWithQuasiIdenticalError),
+        RpcTest::identity(ChainGetTipSetV2::request((TipsetSelector {
+            key: None.into(),
+            height: None,
+            tag: None,
+        },))?)
+        .policy_on_rejected(PolicyOnRejected::PassWithQuasiIdenticalError),
+        RpcTest::identity(ChainGetTipSetV2::request((TipsetSelector {
```

Comment:
Won't this test be flaky? What if we're at the epoch boundary, e.g., 29s and Forest's latest is X and Lotus's is X+1?

Reply:
Nah, this test should fail due to >1 selection criteria.